src/network/constants.rs

```diff
@@ -1,5 +1,10 @@
 //Following module contains all the network constants
 
+// If network name is equivalent to default, netavark knows that this is the default network
+// netavark chooses to do certain things for default network differently
+// E.g Does not configures DNS nameservers for default network
+pub static DEFAULT_NETWORK_NAME: &str = "podman";
```

Comment:
I don't think we want to continue this with the Netavark/Aardvark stack. Even the default network should do DNS.

Reply:
Yeah this is wrong. DNS should work the same for everything. We have a dns_enabled field which says if we have to do dns.

Reply:
ah fair i'll use `dns_enabled` no worries.